lib/graphql/enum_type.rb

```diff
@@ -67,6 +67,7 @@ def validate_non_null_input(value_name)
     # @param value_name [String] the string representation of this enum value
     # @return [Object] the underlying value for this enum value
     def coerce_non_null_input(value_name)
+      return nil unless @values_by_name.keys.include?(value_name)
       @values_by_name.fetch(value_name).value
```

Comment:
https://en.wikipedia.org/wiki/There%27s_more_than_one_way_to_do_it

Sometimes I wish we were using python :trollface: